docs/TOC.md

```diff
@@ -229,9 +229,10 @@
 ## [Autocomplete](api/search-autocomplete-service-resource.md)
 ## [Package metadata](api/registration-base-url-resource.md)
 ## [Package content](api/package-base-address-resource.md)
-## [Report abuse URL](api/report-abuse-resource.md)
 ## [Repository signatures](api/repository-signatures-resource.md)
 ## [Catalog](api/catalog-resource.md)
+## [Report abuse URL](api/report-abuse-resource.md)
```

Comment:
We should create a new node called "Resources" and make all the resource docs as a child in alphabetical order.
Update [this doc](https://docs.microsoft.com/en-us/nuget/api/overview#resources-and-schema) to list the resources in alphabetical order too and potentially link to the resource specific page from this table.

Reply:
Spoke offline.

- Remove code marker around resource names in overview table
- Make header names code on overview and track issue
- Make the Resources folder right after Overview